### DISCUSSION TITLE
Ability to use DB::query()

### DISCUSSION THREAD

**Discussion by ph7jack** [filamentphp/filament#6593] [Help]
Hi, I have a project with so much data in the database, and I need to join many tables to build the query.
How can I use a `Illuminate\Database\Query\Builder` instead of a `Illuminate\Database\Eloquent\Builder`? 
I really need the ability to use a **DB query** instead of using a model.

Related:
https://github.com/filamentphp/filament/discussions/2283

**Comment by danharrin** [comment]
We're considering this for version 3, but it may not happen as the implementation is incredibly complex

**Reply by Ir001** [reply]
any update for this?
